servers/audio/audio_stream.cpp

```diff
@@ -387,9 +387,11 @@ void AudioStreamRandomizer::add_stream(int p_index) {
 	notify_property_list_changed();
 }
 
+// p_index_to is relative to the array prior to the removal of from.
+// Example: [0, 1, 2, 3], move(1, 3) => [0, 2, 1, 3]
 void AudioStreamRandomizer::move_stream(int p_index_from, int p_index_to) {
```

Comment:
Indeed that's how this method works currently. Seems kinda suspicious. :thinking:
@ellenhp Can you please confirm this method is supposed to work like that?
(it's unrelated to this PR)

Reply:
I am currently working on a fix. It's actually an underlying quirk with the way the new editor arrays work.